src/protocol/libp2p/identify.rs

```diff
@@ -356,7 +357,10 @@ impl Identify {
         loop {
             tokio::select! {
                 event = self.service.next() => match event {
-                    None => return,
+                    None => {
+                        tracing::warn!(target: LOG_TARGET, "transport service stream ended, terminating identify event loop");
```

Comment:
Is this a warn because a terminated identify loop is not very expected in normal operation?

Reply:
Yep, we don't expect any of these components to exit under normal circumstances. If they exit however, this should give us enough information about what happened to isolate the issue 🙏